openslides/motions/serializers.py

```diff
@@ -101,12 +101,43 @@ class WorkflowSerializer(ModelSerializer):
     Serializer for motion.models.Workflow objects.
     """
     states = StateSerializer(many=True, read_only=True)
-    first_state = PrimaryKeyRelatedField(read_only=True)
+    # The first_state is checked in the update() method
+    first_state = PrimaryKeyRelatedField(queryset=State.objects.all(), required=False)
 
     class Meta:
         model = Workflow
         fields = ('id', 'name', 'states', 'first_state',)
 
+    @transaction.atomic
+    def create(self, validated_data):
+        """
+        Customized create method. Creating a new workflow does always create a
+        new state which is used as first state.
+        """
+        workflow = super().create(validated_data)
+        first_state = State.objects.create(
+            name='new',
+            action_word='new',
```

Comment:
This should be something like 'set to new' or 'set new'.

Reply:
@normanjaeckel I don't like it. I prefer to remove the action word and use always the state name! Action words are sometimes irritating for users, e.g. if a motion was accepted the use has to click on "accept" (action word) to get "accepted" (state name). But often users think alread in "accepted" if they search for the end state.
What do you think?

Reply:
issue #3776 created.